test/integration-tests/IntegrationTests.Helpers/Comptability/DictionaryExtensions.cs

```diff
@@ -20,7 +20,7 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 
-namespace IntegrationTests.Helpers.Comptability
+namespace IntegrationTests.Helpers.Compatibility
```

Comment:
Please fix also the project structure. For now files are in `Comptability` folder.